README.ORG

```diff
@@ -34,12 +34,13 @@
       git clone https://github.com/agzam/spacehammer ~/.hammerspoon
     #+end_src
 ** LEAD keybinding
-   =LEAD= is the main and major keybinding that invokes main Spacehammer modal. By default set to =Option+SPC=, but it can be re-configured in =~/.spacehammer/config.fnl= by changing the `:mods` and `:key` that has the `"lib.modal:activate-modal"` action. You might want to set it, for example to =Cmd+SPC=. Be warned though, in OS X =Cmd+SPC= is usually used for Spotlight.
```

Comment:
The lib.modal:activate-modal should be surrounded like` ="lb.modal:activate-modal"=` for verbatim values and `~code or command~` for code

Reply:
Nevermind, that was removed! Brain dead this morning 😵